alarm/migrations/0001_initial.py

```diff
@@ -16,21 +16,21 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='PhoneNumber',
+            name='Phone',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('phone_number', phonenumber_field.modelfields.PhoneNumberField(blank=True, max_length=128, region='UA', unique=True)),
-                ('pause_service', models.BooleanField(default=False)),
+                ('number', phonenumber_field.modelfields.PhoneNumberField(blank=True, max_length=128, region='UA', unique=True)),
+                ('enabled', models.BooleanField(default=False)),
                 ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
             ],
         ),
         migrations.CreateModel(
             name='Coin',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('short_name_coin', models.CharField(max_length=255)),
+                ('coin_abbreviation', models.CharField(max_length=255)),
                 ('threshold', models.DecimalField(decimal_places=2, max_digits=10)),
-                ('phone_number', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='alarm.phonenumber')),
+                ('phone', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='alarm.phone')),
             ],
         ),
     ]
```

Comment:
1. Nevere update migrations that already merged to develop or master. You will corrupt another team members DBs. If you need update model, always create a new migration.
2. You updated the migration file but where updated model?